src/Items/ItemEyeOfEnder.h

```diff
@@ -0,0 +1,59 @@
+
+#pragma once
+
+#include "ItemHandler.h"
+#include "ItemThrowable.h"
```

Comment:
Since this class extends cItemThrowableHandler, updating the comment in ItemThrowable.h to include ender pearls would be a good idea

Reply:
Got it.